Adding Launchpad documentation (#4216)
segmentio/segment-docs: src/connections/destinations/catalog/launchpad/index.md

> ## title:Launchpad Destination

Reviewer: ```suggestion
---
title: Launchpad (Actions) Destination
---
```

{% include content/plan-grid.md name="actions" %}

> [<Launchpad>](https://www.launchpad.pm/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners)  is Mission Control for growing product-led companies. It helps you monitor projects and outcomes in one place so that you can keep your business on track..

Reviewer: ```suggestion
[Launchpad](https://www.launchpad.pm/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"} is mission control for growing product-led companies. Launchpad helps you monitor projects and outcomes in one place so that you can keep your business on track.
```

> > (delete after reading) Update your company name and support email address.

Reviewer: ```suggestion
```

> This destination is maintained by <Launchpad>. For any issues with the destination, [contact our Support team](mailto:support@launchpad.pm).

Reviewer: ```suggestion
Launchpad maintains this destination. For any issues with the destination, [contact the Launchpad Support team](mailto:support@launchpad.pm).
```

{% include content/ajs-upgrade.md %}

## Getting started

> 1. From the Segment web app, click **Catalog**, then click **Destinations**.

Reviewer: ```suggestion
1. From the Segment web app, navigate to **Connections > Catalog** and click **Destinations**.
```

> 2. Find the Destinations Actions item in the left navigation, and click it.

Reviewer: ```suggestion
2. Select Destinations Actions under **Categories** from the left navigation.
```

> 3. Click **Configure Launchpad**.

Reviewer: ```suggestion
3. Search for **Launchpad (Actions)** and click **Configure Launchpad**.
```

4. Select an existing Source to connect to Launchpad (Actions).

> 5. Copy your API Key from Launchpad. You can find that in the settings of the app. 

Reviewer: ```suggestion
5. Copy your API Key from Launchpad. You can find the API key in the settings of the Launchpad app. 
```

{% include components/actions-fields.html %}

> This destination is meant to work a standard Segment integration. We work with track, identify and group events to showcase launch performance and help you better understand your customers.

Reviewer: ```suggestion
Launchpad works with track, identify and group events to showcase launch performance and help you better understand your customers.
```

> ---

Reviewer: ```suggestion
```
